hibernate-core/src/main/java/org/hibernate/type/descriptor/jdbc/LongVarbinaryJdbcType.java

```diff
@@ -14,14 +14,19 @@
 public class LongVarbinaryJdbcType extends VarbinaryJdbcType {
 	public static final LongVarbinaryJdbcType INSTANCE = new LongVarbinaryJdbcType();
 
-	private final int jdbcTypeCode;
+	private final int defaultSqlTypeCode;
 
 	public LongVarbinaryJdbcType() {
-		this(Types.LONGVARBINARY);
+		this( Types.LONGVARBINARY );
 	}
 
-	public LongVarbinaryJdbcType(int jdbcTypeCode) {
-		this.jdbcTypeCode = jdbcTypeCode;
+	public LongVarbinaryJdbcType(final int defaultSqlTypeCode) {
```

Comment:
```suggestion
	public LongVarbinaryJdbcType(int defaultSqlTypeCode) {
```

We don't usually bother sticking `final` on parameters, even though it makes sense.